src/containers/navigation/index.tsx

```diff
@@ -0,0 +1,8 @@
+import { createAppContainer, createSwitchNavigator } from 'react-navigation';
+import Auth from '../../screens/Auth';
+
+const AppNavigator = createSwitchNavigator({
+  Auth: { screen: Auth },
```

Comment:
move Auth to the screen constants ( i mean ```key``` ) and use UPPERCASE for all constants